src/plot_api/plot_api.js

```diff
@@ -1374,7 +1374,8 @@ function _restyle(gd, aobj, _traces) {
         'line.cmin', 'line.cmax',
         'marker.line.cmin', 'marker.line.cmax',
         'contours.start', 'contours.end', 'contours.size',
-        'contours.showlines',
+        'contours.showlines', 'contours.showlabels', 'contours.labelformat',
```

Comment:
@etpinard I had a misstep in https://github.com/plotly/plotly.js/pull/1815/commits/e5a2f91e0c5e3e7a25fd463398fbd0322c6dd908 when I thought we could hunt for `editType` inside containers, but your awesome plot_api tests showed that wasn't going to work - so removed that piece in https://github.com/plotly/plotly.js/pull/1815/commits/cbdf795d2c1b1154fd5968855636d163d986aab4 - look reasonable? Another baby step to cleaning up `restyle` and `relayout`...